src/app/modules/webhook/webhook.service.ts

```diff
@@ -2,20 +2,27 @@ import axios, { AxiosError, AxiosResponse } from 'axios'
 import { get } from 'lodash'
 import mongoose from 'mongoose'
 
-import formsgSdk from '../../config/formsg-sdk'
-import { createLoggerWithLabel } from '../../config/logger'
+import formsgSdk from '../../../config/formsg-sdk'
+import { createLoggerWithLabel } from '../../../config/logger'
 // Prevents JSON.stringify error for circular JSONs and BigInts
-import { stringifySafe } from '../../shared/util/stringify-safe'
+import { stringifySafe } from '../../../shared/util/stringify-safe'
 import {
   IFormSchema,
   ISubmissionSchema,
   IWebhookResponse,
-  WebhookParams,
-} from '../../types'
-import { getEncryptSubmissionModel } from '../models/submission.server.model'
-import { WebhookValidationError } from '../modules/webhooks/webhook.errors'
+  WebhookView,
+} from '../../../types'
+import getSubmissionModel, {
+  getEncryptSubmissionModel,
+} from '../../models/submission.server.model'
+
+import { WebhookValidationError } from './webhook.errors'
+import { WebhookParams } from './webhook.types'
+import { validateWebhookUrl } from './webhook.utils'
 
 const logger = createLoggerWithLabel(module)
+// eslint-disable-next-line
```

Comment:
Why is this here?

Reply:
Can be removed now that https://github.com/opengovsg/FormSG/pull/244 has been merged in